src/components/organisms/pupil/OakLessonTopNav/OakLessonTopNav.tsx

```diff
@@ -11,7 +13,7 @@ import {
 import { OakRoundIcon } from "@/components/molecules";
 import { getBreakpoint } from "@/styles/utils/responsiveStyle";
 
-type LessonSectionName = "intro" | "starter-quiz" | "video" | "exit-quiz";
+type LessonTopNavSectionName = Omit<LessonSectionName, "overview" | "review">;
```

Comment:
Storybook was complaining that the lesson section names weren't compatible, so I've used the same sections but omitted `overview` and `review` as that matches the previous definition and I assume the nav isn't displayed on those sections.